src/main/java/com/google/sps/servlets/ActionsServlet.java

```diff
@@ -0,0 +1,66 @@
+package com.google.sps.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.lang.Process;
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import com.google.gson.Gson;
+
+/** Servlet that actions items.*/
+@WebServlet("/actions")
+public class ActionsServlet extends HttpServlet {
+
+  private static final List<String> terms = Arrays.asList("Black Lives Matter", "COVID-19");
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    List<String> jsonResultList = new ArrayList<>();
+    String apiKey = "API_KEY"; //Insert the API_KEY here for testing
```

Comment:
Should this be an instance variable (asking this for myself as well)? Though I guess in the final implementation, the API key won't be in the code anyway.

Reply:
Yeah, I think it should be an instance variable.